docs/05-technical-reference/evnt-01-event-types.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Event types
+---
+
+Eventing supports the following event types:
+- [Cloud Events](https://cloudevents.io/) - they use a specification for describing event data in a common way.
+- legacy events - they are converted to Cloud Events by [Event Publisher Proxy](./00-architecture/evnt-01-architecture.md#event-publisher-proxy), which also adds a `sap.kyma.custom` prefix.
+
+## Event type format
+
+For a Subscription Custom Resource, the fully qualified event name takes the sample form of `sap.kyma.custom.commerce.order.created.v1` or `sap.kyma.custom.commerce.Account.Root.Created.v1`.
+
+The event type is composed of the following components:
+- Prefix: `sap.kyma.custom`
+- Application: `commerce`
+- Event: can have two or more segments separated by `.`; for example, `order.created` or `Account.Root.Created`
+- Version: `v1`
+
+For publishers, the event type takes this sample form:
+- `order.created` or `Account.Root.Created` for legacy events coming from the `commerce` application
+- `sap.kyma.custom.commerce.order.created.v1` or `sap.kyma.custom.commerce.AccountRoot.Created.v1` for Cloud Events
+
+
+## Event name cleanup
```

Comment:
Maybe a better name for the whole document is "Event names"? The types are linked with the names because there is a different name for different type. It's not exactly troubleshooting, just providing information about an automatic process in Eventing (making sure there are no hyphens in event names). I can also split the info into two documents? Thank you for the comment 👍

Reply:
I renamed the document to "Event names", I hope that is okay.